staging/src/k8s.io/client-go/kubernetes/typed/core/v1/fake/fake_pod_expansion.go

```diff
@@ -26,6 +26,7 @@ import (
 func (c *FakePods) Bind(binding *v1.Binding) error {
 	action := core.CreateActionImpl{}
 	action.Verb = "create"
+	action.Namespace = binding.Namespace
```

Comment:
is name needed as well?

Reply:
@liggitt The action is `create`, so the name is not required in action and already in the `binding` object.